src/components/AcceleratorReports/AcceleratorReportsTable.tsx

```diff
@@ -174,34 +187,42 @@ export default function ReportsList({ fromConsole }: ReportsListsProps): JSX.Ele
               id='yearFilter'
               label=''
               onChange={(year: string) => setYearFilter(year)}
-              options={availableYears}
+              options={yearFilterOptions}
               placeholder={strings.YEAR}
               selectedValue={yearFilter}
               sx={{ textAlign: 'left' }}
             />
           </Box>
         </>
       ) : null,
-    [activeLocale, availableYears, yearFilter]
+    [activeLocale, yearFilterOptions, yearFilter]
   );
 
   useEffect(() => {
     if (!yearFilter) {
-      setYearFilter(availableYears[0]);
+      setYearFilter(yearFilterOptions[0]);
     }
-  }, [availableYears, yearFilter]);
+  }, [yearFilterOptions, yearFilter]);
+
+  if (!projectId) {
+    return <></>;
+  }
+
+  if (listAllAcceleratorReportsRequest?.status === 'success' && !allAcceleratorReports?.length) {
```

Comment:
I'm not seeing this message. Maybe the condition should be allAcceleratorReports?.length === 0 ?

If not adding the current year, this wouldn't be necessary because we will always have reports for selected years

Reply:
This change is primarily for the participant side, but just realized I need to add some styles so it doesn't look so crude 😅

![localhost_3000_accelerator_projects_8_reports_tab=reports](https://github.com/user-attachments/assets/fe596b01-320c-4a22-98c1-e421239aa0b0)

Reply:
Went ahead and removed this message until we have designs for the empty state.